Package.swift

```diff
@@ -7,7 +7,7 @@ let package = Package(
       .library(name: "StencilSwiftKit", targets: ["StencilSwiftKit"])
   ],
   dependencies: [
-      .package(url: "https://github.com/stencilproject/Stencil.git", .upToNextMinor(from: "0.13.0"))
+      .package(url: "https://github.com/stencilproject/Stencil.git", .branch("master"))
```

Comment:
I'd rather avoid this if possible, I can probably push through a Stencil release if needed.

Do we have Swift 5 support in Stencil yet? (it's been a while)